packages/web/index.js

```diff
@@ -219,17 +218,6 @@ module.exports = (neutrino, opts = {}) => {
       config.devtool('cheap-module-eval-source-map');
       config.when(options.hot, () => {
         neutrino.use(hot);
-        config.when(options.hotEntries, (config) => {
```

Comment:
I think we should add a check for `hotEntries` so the user knows to clean up their config (and also why things aren't working if they rely on it) - like we do for the deprecated `options.minify` options above.

Reply:
Good call.